api/v1beta1/neutronapi_types.go

```diff
@@ -194,6 +198,13 @@ type NeutronAPIDebug struct {
 	Service bool `json:"service"`
 }
 
+// NeutronRouteAnnotations defines the route annotations
+type NeutronRouteAnnotations struct {
+	// +kubebuilder:validation:Optional
+	// Timeout - Provides the HAProxy Timeout value for the route
+	Timeout string `json:"timeout"`
```

Comment:
should we perhaps also choose a sane default value here? (120s?)

Reply:
I’m not sure we should have a default in Neutron-operator. It would be more in line with user expectations that if they don’t provide a value here then the default will be OpenShift route defaults.

If we add a default here, we might cause some confusion for users.

Reply:
I think there could be argument here to increase the default... by default. This PR is for editing the server timeout [1] and just taking a quick look at one of the OpenStack labs we have the same value set by default to 10m for Neutron [2]

[1] https://github.com/openshift/router/blob/0cd9a58dff77127562f7851ccfa5cda79b8cef48/images/router/haproxy/conf/haproxy-config.template#L516
[2] timeout server 10m

Reply:
We discussed the matter of default choice networking team call just now, and the assumption is that we will retain whatever the default value we have for the same in tripleo, which is 120s at the moment. There was a legitimate reason to bump the value in tripleo for neutron api that we may dig for, but I don't have it at the moment.

That said, I don't think this is required that this PR addresses the choice of the default value. This can be done in a separate PR, if needed.